presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -295,4 +638,32 @@ private boolean outerJoinCurrentPosition(LookupSource lookupSource)
         }
         return true;
     }
+
+    // This class needs to be public because LookupJoinOperator is isolated.
```

Comment:
s/needs/must

Reply:
That was a verbatim copy from `Counter` class ;)

(applied anyway)